### DISCUSSION TITLE
How to start a script on boot-up?

### DISCUSSION THREAD

**Discussion by AIWildUItraman** [ifm/ifm3d#410] [General]
@dekhanra I want to automatically run a script after the device boots up, but I don't have administrator privileges. What should I do?

**Comment by graugans** [comment]
Hej @AIWildUItraman this is a quite unspecific question. What ifm device and are you asking for. I guess the an  OVP8xx is the device you asking for. It is not recommended to run a script on the OVP8xx please use a Docker container instead. There is no defined environment we can gurantee for the tooling and the version of system libraies. This is why we recommend Docker on the OVP8xx please check the [documentation](https://ifm3d.com/documentation/SoftwareInterfaces/Docker/autostart.html#autostart-a-container-on-the-vpu).

**Reply by AIWildUItraman** [reply]
ok! I got it!! Thanks!
